stable/graylog/README.md

```diff
@@ -96,56 +96,57 @@ graylog:
 
 The following table lists the configurable parameters of the Cassandra chart and their default values.
```

Comment:
Please fix the copied & paste text from `Cassandra`. And also avoid to reformat entire file. It is hard to compare and review.

Reply:
Done